util/testutil/integration/util_windows.go

```diff
@@ -11,6 +11,12 @@ import (
 
 var socketScheme = "npipe://"
 
+var windowsImagesMirrorMap = map[string]string{
+	"nanoserver:latest": "mcr.microsoft.com/windows/nanoserver:ltsc2022",
+	"servercore:latest": "mcr.microsoft.com/windows/servercore:ltsc2022",
+	"busybox:latest":    "registry.k8s.io/e2e-test-images/busybox:1.29-2",
```

Comment:
Does the current code mirror all architectures of busybox? If you look at linux version then you see that we have some special handling for that.

This would also impact the other images if they ever get `arm64` versions.

If this is issue for busybox atm you can just link the amd64-only reference here directly and we can deal with this again when there are windows-arm64 variants.

Reply:
No, it's just arm64. The image itself also only has one arch. Let me put a TODO for that.

Reply:
https://explore.ggcr.dev/?image=registry.k8s.io%2Fe2e-test-images%2Fbusybox%3A1.29-2 looks multi-arch

Reply:
missed that, I had just inspected what I had pulled down. Noted.

Reply:
Didn't notice updates for this?

Reply:
I put a TODO above on L15 to revisit this. We are currently just using nanoserver amd64 images for our tests.

Reply:
The issue was with the busybox, that was multi-platform.

Reply:
Can I just drop it for now?

Reply:
Just tag it to the amd64 image so the time is not spent mirroring other platforms.

Reply:
They don't have an amd64 specific tag, I'll just pick the sha256 from the manifest.
```
$ crane ls registry.k8s.io/e2e-test-images/busybox
1.29
1.29-1
1.29-2
1.29-4
1.36.1-1
```

Reply:
However, on Windows, as part of the image pull, the platform matcher picks the right image based on the architecture and the platform OS version.

Eg.
```
=== NAME  TestIntegration
    run.go:272: copied mcr.microsoft.com/windows/nanoserver:ltsc2022 to local mirror localhost:65513/library/nanoserver:latest
time="2024-07-25T23:04:21+03:00" level=info msg="trying next host - response was http.StatusNotFound" host="localhost:65513"
    run.go:272: copied registry.k8s.io/e2e-test-images/busybox:1.29-2 to local mirror localhost:65513/library/busybox:latest
````

Will it be necessary to provide the specific sha256?

Reply:
you should be able to just pin to the digest.

> However, on Windows, as part of the image pull, the platform matcher picks the right image based on the architecture and the platform OS version.

The puller will pull the correct one, but mirror I think will copy all architectures to local registry before.

Reply:
I see, sure, I've pinned to the digest of the latest image (`busybox:1.36.1-1`) based on `ltsc2022`.